test/load/traders/random-trader.js

```diff
@@ -0,0 +1,47 @@
+const Promish = require('promish');
+const Trader = require('./trader');
+const strategyConfig = require('../random-strategy.json');
+
+class RandomTrader extends Trader {
+
+  start() {
+    this.isActive = true;
+    this.marketPrice = strategyConfig.price;
+    const loop = () => {
+      if (!this.isActive) {
+        this.promise = Promish.resolve();
+      } else {
+        let random = Math.round(Math.random(), 2);
```

Comment:
Missed this one last time - Math.random() returns a floating point number between 0 & 1 and Math.round() does not support precision so this line above generates 0 or 1 but nothing in between.
This would be better
```
const random = Math.random() * 2 - 1; // generate random number from -1 to 1
```